### PR TITLE
Adds support for LTI Dynamic Registration

### PR DESCRIPTION
**Pull request by jbasdf**
Adds support for the IMS LTI Dynamic Registration specification:
https://www.imsglobal.org/spec/lti-dr/v1p0

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@
   resources :public_keys
   resource :lti_config
 
+  resources :lti_dynamic_registrations
```
Comment (bfcoder) [inline config/routes.rb:12]: Only :index is in the controller, should we restrict this? Or will downstream be adding more?

Comment (snapbrains) [reply config/routes.rb:12]: I think we'll at least need :show at some point. I'm inclined to leave it for now. Is there any risk in leaving it?

Comment (jbasdf) [reply config/routes.rb:12]: I think we leave it for now. It is likely that we will need additional endpoints in the downstream apps.

Comment (bfcoder) [reply config/routes.rb:12]: ok 👍

**Review by bfcoder — APPROVED** [review]

**Comment by bfcoder** [comment]
Looks good

**Pull request merged**